gmso/parameterization/topology_parameterizer.py

```diff
@@ -58,44 +58,24 @@ class TopologyParameterizationConfig(GMSOBase):
         "angle, dihedral, improper etc",
     )
 
-    identify_connected_components: bool = Field(
+    speedup_by_molgraph: bool = Field(
         default=False,
         description="A flag to determine whether or not to search the topology"
         " for repeated disconnected structures, otherwise known as "
         "molecules and type each molecule only once.",
     )
 
-    use_molecule_info: bool = Field(
+    speedup_by_moltag: bool = Field(
         default=False,
         description="A flag to determine whether or not to look at site.molecule "
         "to look parameterize each molecule only once. Will only be used if "
-        "identify_connected_components=False",
-    )  # Unused
-
-    assert_bond_params: bool = Field(
-        default=True,
-        description="If True, an error is raised if parameters are not found for "
-        "all system bonds.",
+        "speedup_by_molgraph=False",
```

Comment:
```suggestion
        "speedup_by_molgraph=True",
```

Reply:
Shouldn't this be False? The parameter is set to False by default, which was kind of the original point of this PR. Or am I missing something?

Reply:
Yes, but that's not what this bit of text is saying. Here's the whole text in context:
``` speedup_by_moltag: bool = Field(
        default=False,
        description="A flag to determine whether or not to look at site.molecule "
        "to look parameterize each molecule only once. Will only be used if "
        "speedup_by_molgraph=False",
```

So that last line should read speedup_by_molgraph=True, since that's when it's used.

Reply:
ahh ok I see what you mean.